deploy-kubernetes.yml

```diff
@@ -2,11 +2,15 @@
 - hosts: all
   remote_user: root
   vars_files:
-    - "{{ playbook_dir }}/kubernetes/vars/default_vars.yml"
-    - "{{ playbook_dir }}/common/vars/default_vars.yml"
+  pre_tasks:
+    - name: override roles variables
+      include_vars: "{{ item }}"
+      with_items:
+        - "{{ playbook_dir }}/kubernetes/vars/default_vars.yml"
+        - "{{ playbook_dir }}/common/vars/default_vars.yml"
   roles:
     - "{{ playbook_dir }}/kubernetes/roles/prerequisites"
-    - "{{ playbook_dir }}/common/roles/golang"
```

Comment:
Is there also a fix upstream against this role?

Reply:
This role was removed from this repository by https://github.com/cynepco3hahue/kubevirt-ansible/pull/9/files
But unfortunately it wasn't removed from `deploy-kubernetes.yml` file.

Does it answer your Q? If it is not let me know :-)